gprofiler/profilers/java.py

```diff
@@ -274,6 +276,37 @@ def get_ap_version() -> str:
 T = TypeVar("T", bound="AsyncProfiledProcess")
 
 
+class ProcessUptimeCalculator:
+    DEFAULT_BINS = [5, 10, 30, 60, 300]
+
+    def __init__(self, process: Process):
+        self._create_time: Optional[float]
+        self._process = process
+        try:
+            self._create_time = process.create_time()
+        except psutil.NoSuchProcess:
+            self._create_time = None
+
+    def get_alive_time(self) -> Optional[float]:
+        if self._create_time is None:
+            return None
+        return time.time() - self._create_time
```

Comment:
Seems like psutil works with local time as well, any way to force it to work with absolute timestamps (and use `time.monotonic` as well, such that we're not affected by clock changes?)